foundation-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -95,7 +97,32 @@ MobileCore.configureWithAppId("1423ae38-8385-8963-8693-28375403491d");
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
```

Comment:
Let's reverse the order here and have iOS (AEP 3.x) then iOS (ACP 2.x) - this applies for all the sections below